tools/pipelines/pipeline.static-site.js

```diff
@@ -185,15 +188,15 @@ module.exports = function setupNunjucksPagesPipeline(gulp) {
                   });
               }
 
-              if(contentFile) {
+              if(contentFile && !pageOptions.lastModDate) {
                 contentPath = contentBase.concat('/', contentFile)
 
                 await fetch(apiPath(contentPath), {
                   headers: { 'Authorization': `Bearer ${accessToken}` }
                 })
                 .then(data => data.json())
                 .then((commit) => {
-                  lastModDates.push(commit[0].commit.committer.date);
+                  lastModDates.push(new Date(commit[0].commit.committer.date));
```

Comment:
Im not really familiar with this change, what is this doing for the generator?

Reply:
We're comparing the dates inside lastModDates to get the most recent one. Before we were just comparing the date strings we get from Github. The new lastModDate that is passed through is a date object, so we convert these to actual dates to compare that instead.